### PR TITLE
Modify 3PlayMedia turnaround levels 

### PR DESCRIPTION
**Pull request by mushtaqak**
[EDUCATOR-1809](https://openedx.atlassian.net/browse/EDUCATOR-1809)

Related pipeline PR: https://github.com/edx/edx-video-pipeline/pull/72
Related platform PR: https://github.com/edx/edx-platform/pull/16672

[Sandbox](https://studio-pr16672.sandbox.edx.org/videos/course-v1:edX+DemoX+Demo_Course)

### CODE REVIEW

**Comment by mushtaqak** [comment]
@muhammad-ammar @Qubad786 This is ready for your review

File: edxval/utils.py [hunk 1]
```diff
@@ -59,11 +59,12 @@
...
         'display_name': '3Play Media',
         'turnaround': {
-            'same_day_service': 'Same day',
-            'rush_service': '24 hours (rush)',
-            'expedited_service': '2 days (expedited)',
+            'two_hour': '2 hours',
+            'same_day': 'Same day',
+            'rush': '24 hours (rush)',
+            'expedited': '2 days (expedited)',
             'default': '4 days (default)',
```
Comment (Qubad786) [inline edxval/utils.py]: defaults to `standard`

File: edxval/models.py [hunk 2]
```diff
@@ -512,18 +512,20 @@ class ThreePlayTurnaround(object):
...
-    SAME_DAY_SERVICE = 'same_day_service'
+    EXPEDITED= 'expedited'
+    RUSH = 'rush'
+    SAME_DAY= 'same_day'
+    TWO_HOUR = 'two_hour'
 
     CHOICES = (
-        (EXTENDED_SERVICE, '10-Day/Extended'),
+        (EXTENDED, '10-Day/Extended'),
         (DEFAULT, '4-Day/Default'),
```
Comment (Qubad786) [inline edxval/models.py]: defaults to `standard`

**Comment by mushtaqak** [comment]
@Qubad786 Addressed feedback

**Pull request merged**